polyglot/piranha/src/models/piranha_arguments.rs

```diff
@@ -12,140 +12,175 @@ Copyright (c) 2022 Uber Technologies, Inc.
 */
 
 use clap::Parser;
-use colored::Colorize;
+
+use crate::{utilities::read_toml};
 use derive_builder::Builder;
 use getset::{CopyGetters, Getters};
-use log::info;
 use std::{collections::HashMap, path::PathBuf};
-use tree_sitter::Language;
+use serde_derive::Deserialize;
+
+use super::{default_configs::{
+  default_cleanup_comments_buffer, default_delete_file_if_empty, default_dry_run,
+  default_global_tag_prefix, default_number_of_ancestors_in_parent_scope, default_substitutions, default_languages, default_cleanup_comments,
+}, language::{PiranhaLanguage, get_language}};
 
-use crate::{
-  config::CommandLineArguments,
-  models::piranha_config::PiranhaConfiguration,
-  utilities::{read_toml, tree_sitter_utilities::TreeSitterHelpers},
-};
 
-#[derive(Clone, Builder, Getters, CopyGetters, Debug)]
-#[builder(default)]
+pub struct CommandLineInput {}
+
+#[derive(Deserialize, Clone, Builder, Getters, CopyGetters, Debug, Parser, Default)]
+#[clap(name = "Piranha1")]
 /// Captures the processed Piranha arguments (Piranha-Configuration) parsed from `path_to_feature_flag_rules`.
 pub struct PiranhaArguments {
   /// Path to source code folder.
-  #[getset(get = "pub")]
+  #[get = "pub"]
```

Comment:
`#[get = "pub"]` -> Macro for public getter 
 `#[builder(default = "String::new()")]` -> macro for builder pattern setter, with default value 
`#[clap(short = 'c', long = "path_to_code_base")]` -> macro for indicating that this prop should be made available as a command line argument 
 `#[serde(skip)]` -> skip during `serde`